runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -176,6 +176,13 @@ public class LootTrackerPlugin extends Plugin
 		put(13117, "Rogues' Chest").
 		build();
 
+	// Chests opened with keys from slayer tasks
+	private static final Set<String> SLAYER_CHEST_EVENT_TYPES = new ImmutableSet.Builder<String>().
+		add("Brimstone Chest").
+		add("Larran's big chest").
+		add("Larran's small chest").
+		build();
```

Comment:
```suggestion
	private static final Set<String> SLAYER_CHEST_EVENT_TYPES = ImmutableSet.of(
		"Brimstone Chest",
		"Larran's big chest",
		"Larran's small chest"
	);
```

Reply:
Changed in b8175f742a5d8c62bd439918800fcf0b8a97015b, thanks.